src/frontend/src/test-e2e/util.ts

```diff
@@ -48,6 +48,7 @@ export async function runInBrowserCommon(
       "goog:chromeOptions": {
         args: [
           "--headless",
+          "--ignore-certificate-errors",
```

Comment:
Can you add a note about why this is needed? Also I reckon this is only needed/wanted in the github actions runs, not locally

Reply:
No, this is always required, because we use self-signed certificates.